ui/widgets/resizable.js

```diff
@@ -226,6 +226,9 @@ $.widget( "ui.resizable", $.ui.mouse, {
 			this._removeHandles();
 			this._setupHandles();
 			break;
+		case "aspectRatio":
+			this._aspectRatio = !!value;
```

Comment:
We have two properties `this.aspectRatio` and `this._aspectRatio`.
In aspectRatio we have numbers are used to define the ratio. It is set in `_mouseStart` method https://github.com/jquery/jquery-ui/blob/1.12.1/ui/widgets/resizable.js#L397. 
We have numeric `aspectRatio` even in cases when we do not set  aspectRatio in options, because we must know it when resize with `shiftKey` pressed.
We use `_aspectRatio` to know save aspect ratio without pressed `shiftKey` (anytime) or not.